defaults/main.yml

```diff
@@ -8,6 +8,10 @@ project: xtesting
 db_project: "{{ project }}"
 use_slave:  # true if slaves are deployed
 user_mail: root@localhost
+use_proxy: false
+
+# control if install software package and services
+deploy_base_infra: true
```

Comment:
At firt glance, I would prefer false by default.
As for privileges, I would prefer the safer way by default (the user does the operations by hand)